Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -30,189 +30,180 @@
+});
+
+addStyle('Sqrt', {
+    color : "color() * sqrt(${temperature})"
+});
+
+addStyle('Sign', {
+    color : "rgb(sign(${POSITION}[0]) * 255, sign(${POSITION}[1]) * 255, sign(${POSITION}[2]) * 255)"
+});
+
+addStyle('Rounding Functions', {
+    color : "rgb(floor(${POSITION}[0]) * 255, ceil(${POSITION}[1]) * 255, round(${POSITION}[2]) * 255)"
+});
+
+addStyle('Exp and Log Functions', {
+    color : "rgb(log(${POSITION}[0]) * 255, log2(${POSITION}[1]) * 255 + exp2(${POSITION}[1]) * 255, exp(${POSITION}[2]) * 255)"
+});
+
+addStyle('Fractional Part', {
+    color : "rgb(fract(${POSITION}[0]) * 255, fract(${POSITION}[1]) * 255, fract(${POSITION}[2]) * 255)"
+});
+
+addStyle('Pow', {
+    color : "color() * pow(${temperature}, 3)"
+});
+
+addStyle('Min and Max', {
+    color : "rgb(min(${POSITION}.x, 0.75) * 255, max(${POSITION}.z, 0.25) * 255, 255)"
+});
+
+addStyle('Clamp and Mix', {
+    color : "color() * clamp(${temperature}, 0.1, 0.2)"
+});
+
+addStyle('Secondary Color', {
+    color : {
+        conditions : [
+            ["${id} < 250", "vec4(${secondaryColor}, 1.0)"],
+            ["${id} < 500", "vec4(${secondaryColor} * ${secondaryColor}, 1.0)"],
+            ["${id} < 750", "vec4(${secondaryColor} / 5.0, 1.0)"],
+            ["${id} < 1000", "rgb(0, 0, Number(${secondaryColor}.x < 0.5) * 255)"]
+        ]
     }
+});
+
+addStyle('Use point colors', {
+    color : "${COLOR} * ${temperature} + rgb(128,128,128)"
+});
+
+addStyle('Use point positions', {
+    show : "${POSITION}[0] > 0.5 || ${POSITION}[1] > 0.5 || ${POSITION}[2] > 0.5"
+});
+
+// POSITION contains 0 as its last component, so add 1.0 to make the point cloud opaque
+addStyle('Color based on position', {
+    color : "vec4(${POSITION}, 1.0)"
+});
+
+addStyle('Style point size', {
+    color : "color('red')",
+    pointSize : "${temperature} * 10"
+});
+
+addStyle('Multiple defines', {
+    defines : {
+        length : "length(${POSITION})",
+        time : "${tiles3d_tileset_time} * 3.0"
+    },
+    color : {
+        conditions : [
+            ["${length} < 0.5", "${length} * color('red')"],
+            ["${length} < 1.0", "vec4(vec3(${temperature} * fract(${time})), 1.0)"],
+            ["true", "${COLOR}"]
+        ]
+    },
+    pointSize : "5.0 - ${length} * 2.0",
+    show : "${length} < 2.0"
+});
+
+function setStyle(style) {
+    return function() {
+        tileset.style = new Cesium.Cesium3DTileStyle(style);
+    };
+}
 
-    Sandcastle.addToolbarMenu(styleOptions);
-    tileset.style = new Cesium.Cesium3DTileStyle(styles[0].style);
+var styleOptions = [];
+for (var i = 0; i < styles.length; ++i) {
+    var style = styles[i];
+    styleOptions.push({
+        text : style.name,
+        onselect : setStyle(style.style)
+    });
 }
 
+Sandcastle.addToolbarMenu(styleOptions);
+tileset.style = new Cesium.Cesium3DTileStyle(styles[0].style);
```

Comment:
Now that the code is shifted back to normal `tileset.style = new Cesium.Cesium3DTileStyle(styles[0].style);` can be removed.